docs/01_introduction/README.md

```diff
@@ -252,25 +253,25 @@ which is reflected in some of the design choices that were made.
 
 ### *Performance*
 
-We strongly prefer **building software from source**, whenever possible.
+EasyBuild strongly prefers to **build software from source code**, whenever possible.
 
 This is important to ensure that the binaries that are installed can maximally exploit
-the capabilities of processors on which the software will be used.
+the capabilities of the machine architecture on which the software will be run.
 
 For that same reason, EasyBuild **optimizes software for the processor architecture of the build host**
 by default, via compiler options like ``-march=native`` (GCC), ``-xHost`` (Intel compilers), etc.
-This behaviour [can be changed via the ``--optarch`` configuration setting](https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html), if desired.
+This behaviour [may be changed via the ``--optarch`` configuration setting](https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html).
 
 
 ### *Reproducibility*
 
-In addition to performance, **reproducibility of installations** is another major focus point.
+In addition to performance, **reproducibility of installations** is vital for EasyBuild.
 
-Most software installations done with EasyBuild use a **particular <a href="#toolchains">toolchain</a>**,
-with which we aim to take control over the build environment and avoid relying on tools & libraries
-provided by the OS. For similar reasons, we try to **provide all required dependencies through EasyBuild** as well,
-with a few notable exceptions (like ``OpenSSL`` for security reasons, and Infiniband and GPU drivers which
-are too closely intertwined with the OS).
+Most software installations performed with EasyBuild use a **particular <a href="#toolchains">toolchain</a>**,
```

Comment:
I deliberately avoided using "performed" because it's too close to "performance", but it does sound better than "done" :), so OK.

Reply:
Ah, yeah. Heh.